roottest/root/tree/fastcloningeventtree/dt_RunDrawTest.C

```diff
@@ -182,22 +172,25 @@ bool dt_RunDrawTest(const char* from, Int_t mode = 0, Int_t verboseLevel = 0) {
 
    TBranch *eb = tree->GetBranch("event");
    gBranchStyle = (int) eb->InheritsFrom(TBranchElement::Class());
-   // cerr << "Branch style is " << gBranchStyle << endl;
+   // std::cerr << "Branch style is " << gBranchStyle << std::endl;
 
-   if (gQuietLevel<2) cout << "Generating histograms from TTree::Draw" << endl;
-   TDirectory* where = GenerateDrawHist(tree,gQuietLevel);
+   if (gQuietLevel<2)
+      std::cout << "Generating histograms from TTree::Draw" << std::endl;
+   TDirectory* where = GenerateDrawHist(tree,2,gQuietLevel);
 
-   if (gQuietLevel<2) cout << "Comparing histograms" << endl;
+   if (gQuietLevel<2)
+      std::cout << "Comparing histograms" << std::endl;
    if (Compare(where)>0) {
-     cout << "DrawTest: Comparison failed" << endl;
-     return false;
+     std::cout << "DrawTest: Comparison failed" << std::endl;
+     return 1;
    }
    DrawMarks();
 
-   if (gQuietLevel<2) cout << "DrawTest: Comparison was successfull" << endl;
+   if (gQuietLevel<2)
+      std::cout << "DrawTest: Comparison was successfull" << std::endl;
    if (hfile) delete hfile;
-   else delete tree;
+         else delete tree;
```

Comment:
(minor) The indentation looks wrong on 'my' github display (it should the `else` starting at the same column as the 'i' in `hfile`.